Source/Scene/PropertyTextureProperty.js

```diff
@@ -96,6 +163,72 @@ Object.defineProperties(PropertyTextureProperty.prototype, {
   },
 });
 
+PropertyTextureProperty.prototype.isGpuCompatible = function () {
```

Comment:
It feels a little out of place to include GPU/GLSL related functionality in `PropertyTextureProperty`. Should it go in the stage instead instead?

Reply:
I've been on the fence about where this should go. These details are very property texture-specific (and furthermore property textures aren't much use on the CPU anyway), property tables would have their own packing/unpacking rules. Didn't want to clutter the MetadataPipelineStage too much, since that already has plenty to manage with attributes, textures and eventually property textures.

Reply:
:thinking: in the future it might be worth splitting such GPU packing details into their own modules or something, though it's hard to tell at this point if that's a good design.

Reply:
Ok, I don't feel too strongly about this, so fine to keep here.